packages/presentation/lib/screen/home/home_tab_bar.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:domain/core/extension/title_extension.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:presentation/core/theme/theme_app.dart';
+import 'package:presentation/screen/home/home_page.dart';
+
+final List<Tab> _myMainTabs = <Tab>[
+  Tab(text: "Card ES", icon: SvgPicture.asset('assets/svg/tab3.svg')),
```

Comment:
Also, svg path better use from constant file;

Reply:
yes it is good practice when you work with real project

for 3 svg paths I didn't use but if they are more should they write in the separated file